astropy/coordinates/baseframe.py

```diff
@@ -632,6 +632,30 @@ def get_representation_component_units(self, which='base'):
 
     representation_component_units = property(get_representation_component_units)
 
+    def _replicate(self, data, copy=False, **kwargs):
```

Comment:
I know this is private, but it might be useful to provide a docstring with parameters here, given how convoluted this logic can get...?